.github/actions/sphinx/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM python:slim
+LABEL maintainer="OpenShot Developers <https://github.com/OpenShot/openshot-qt/>"
+
+WORKDIR /
+RUN apt-get update \
+ && apt-get install --no-install-recommends -y \
+      imagemagick \
+      make \
+      libcairo2 \
+      librsvg2-bin \
+      \
+      latexmk \
+      lmodern \
+      fonts-freefont-otf \
+      texlive-latex-recommended \
+      texlive-fonts-recommended \
+      texlive-luatex \
+      texlive-xetex \
+      xindy \
+      tex-gyre \
+ && apt-get autoremove \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+#      graphviz \
+#      texlive-latex-extra \
+#      texlive-fonts-extra \
+
+COPY ./requirements.txt requirements.txt
```

Comment:
_Note: I'm still working on this a bit, but it's not a blocker to merging_

I'm not entirely happy with the constraint that the `requirements.txt` file for the docs has to live in `.github/actions/sphinx/` now. (The one in `docs/` only contains a reference to that file, and a bunch of comment lines explaining that requirements have to be added to that other file instead of the obvious one. Lame.)

Problem is, the container image fails to build if it tries to reference any files outside the 'build context'. It seems that context is established before the start of the run. Even if I copy the file in, it isn't used. (That's probably a safety measure to avoid script injection, which isn't really the worst idea TBH.)

Trying to find a way I can keep the file in `docs/` while still using it to configure the Docker image. But if I find a solution it can always be a follow-up PR. This PR works as-is for now.